aptos-move/aptos-vm/src/move_vm_ext/session/user_transaction_sessions/abort_hook.rs

```diff
@@ -39,6 +39,12 @@ impl<'r, 'l> AbortHookSession<'r, 'l> {
 
     pub fn finish(self, change_set_configs: &ChangeSetConfigs) -> Result<VMChangeSet, VMStatus> {
         let Self { session } = self;
-        session.finish_with_squashed_change_set(change_set_configs, false)
+
+        // TODO(George): Abort hook can never publish modules! When we move publishing outside
+        //               of MoveVM, we do not need to use _ here, as modules will only be visible
+        //               in user session.
+        let (change_set, _) = session.finish_with_squashed_change_set(change_set_configs, false)?;
```

Comment:
nit: asserting its emptiness would be nice

might be a good reason to have `struct ModuleChangeSet` and `ModuleChangeSet::ensure_empty()`

Reply:
Good point, I will remove this _ in the next few PRs, but let's have an invariant check here